server/templates/obslist.html

```diff
@@ -103,6 +103,17 @@ <h3>Observations</h3>
                     {{ obs.station_name }}
                 </a>
             </td>
+            <td>
+                {% if obs.config.freq is defined %}
+                {{ "%.3f MHz"|format(obs.config.freq) }}
```

Comment:
Could you document that we expect the frequency in megahertz? Maybe, we should use just the hertz and use a template filter to convert the unit? I think that it will be the simplest and best solution.

Reply:
The docs now say it's in MHz. If you want, I can change it to Hz.

Reply:
After second though (and looking at what we use in the station's code), I agree that using Hz is the way to go. The observations list looks the same as before, but the underlying data is in Hz. You can see that when looking at specific observation page.

Reply:
Since I did exactly what was requested, I suppose it's ok to resolve.